libraries/AC_AttitudeControl/AC_AttitudeControl_Heli.h

```diff
@@ -129,8 +141,8 @@ class AC_AttitudeControl_Heli : public AC_AttitudeControl {
     //
     
     // pass through for roll and pitch
-    float _passthrough_roll;
-    float _passthrough_pitch;
+    float _passthrough_roll_cd;
+    float _passthrough_pitch_cd;
 
     // pass through for yaw if tail_passthrough is set
     float _passthrough_yaw;
```

Comment:
This one seems to be in cd too.
```suggestion
    float _passthrough_yaw_cd;
```